gap/AutoDocMainFunction.gi

```diff
@@ -456,9 +474,8 @@ function( pkgname, pkgdir, docdir, main, files, opt )
         fi;
         # pad s to number_of_digits
         s := Concatenation( ListWithIdenticalEntries( number_of_digits - Length( s ), '0' ), s );
-        basename := Concatenation( LowercaseString(pkgname), s, ".tst" );
-        basename := ReplacedString( basename, " ", "_" );
```

Comment:
I think this was not doing anything, not sure why it was even there (package names are not allowed to contain spaces)

Reply:
Ah I was wrong, this can be almost anything for worksheets. But then the old code was kinda wrong, too. Will fix it.